process_report/invoices/pi_specific_invoice.py

```diff
@@ -120,20 +121,17 @@ def _create_html_invoice(temp_fd):
             temp_fd.flush()
 
         def _create_pdf_invoice(temp_fd_name):
-            chrome_binary_location = os.environ.get(
-                "CHROME_BIN_PATH", "/usr/bin/chromium"
-            )
-            if not os.path.exists(chrome_binary_location):
+            if not os.path.exists(CHROME_BINARY_LOCATION):
                 sys.exit(
-                    f"Chrome binary does not exist at {chrome_binary_location}. Make sure the env var CHROME_BIN_PATH is set correctly and that Google Chrome is installed"
+                    f"Chrome binary does not exist at {CHROME_BINARY_LOCATION}. Make sure the env var CHROME_BIN_PATH is set correctly and that Google Chrome is installed"
                 )
```

Comment:
it'd be nice if we called the python constant and the env variable the same - CHROME_BINARY_LOCATION vs CHROME_BIN_PATH.

Reply:
That seems reasonable.